panacea/grpc_client.go

```diff
@@ -15,13 +15,21 @@ import (
 	"google.golang.org/grpc/credentials"
 )
 
-type GRPCClient struct {
+type GRPCClient interface {
+	Close() error
+	BroadcastTx(txBytes []byte) (*tx.BroadcastTxResponse, error)
+	GetCdc() *codec.ProtoCodec
+	GetChainID() string
+	GetAccount(address string) (authtypes.AccountI, error)
+}
+
```

Comment:
Same here.
Let's add `var _ GRPCClient = &gprcClient{}`

Reply:
OK I fixed it(https://github.com/medibloc/panacea-oracle/pull/90/commits/608e1f47c232c695a81da5f8fbe3b0ec750947b8)